src/SFA.DAS.ASK.Web/Controllers/DeliveryPartner/PlanningMeetings/SchedulePlanningMeetingController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.GetOrCreatePlanningMeeting;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.UpdatePlanningMeeeting;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.GetSupportRequest;
+using SFA.DAS.ASK.Web.Infrastructure.ModelStateTransfer;
+using SFA.DAS.ASK.Web.ViewModels.DeliveryPartner.PlanningMeetings;
+
+namespace SFA.DAS.ASK.Web.Controllers.DeliveryPartner.PlanningMeetings
+{
+    public class SchedulePlanningMeetingController : Controller
+    {
+        private readonly IMediator _mediator;
+
+        public SchedulePlanningMeetingController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpGet("delivery-partner/planning-meeting/schedule-planning-meeting/{supportId}")]
+        [ImportModelState]
+        public async Task<IActionResult> Index(Guid supportId, bool edit)
+        
+        {
+            var meeting = await _mediator.Send(new GetPlanningMeetingRequest(supportId));
```

Comment:
As well as a check here that the SupportRequest has the correct status, it will probably be a good idea to check that the logged in user is actually valid for this SupportRequest.